docs/source/using/addons.adoc

```diff
@@ -76,6 +78,12 @@ If the metadata field *OpenShift-Version* is not specified in the add-on header,
 *OpenShift-Version* only supports versions in the form of <major>.<minor>.<patch>.
 ====
 
+[[addon-depends-on]]
+== Add-on Dependency
+
+In the add-on metadata section you can specify dependencies for an add-on. If a certain add-on requires one more other add-ons to be installed, this can be
```

Comment:
Each sentence on its own line, please. :)

I'd also recommend a rewrite like the following:

```adoc
Add-on dependencies can be defined using the _Depends-On_ metadata field.
Multiple dependencies may be defined using a comma-separated list of add-on names.
```

Also, should we include an example here? We shouldn't rely on readers looking back up at the example add-on definition.

Reply:
@robin-owen Removed the example from the original `anyuid` add-on and added an example for only `Depends-On` in its own section.